Protocol.md

```diff
@@ -601,17 +613,21 @@ The message SHALL NOT be encrypted.
 
 After the 'client-hello' message has been sent (responder) or after the
 'server-hello' message has been received (initiator) the client MUST
-send this message to the server. The client MUST set the *your_cookie*
-field to the cookie the server has used in the nonce of the
-'server-hello' message. It SHALL also set the *subprotocols* field to
-the exact same `Array` of subprotocol strings it has provided to the
-WebSocket client implementation for subprotocol negotiation. If the
-user application requests to be pinged (see
-[RFC 6455 section 5.5.2](https://tools.ietf.org/html/rfc6455#section-5.5.2))
-in a specific interval, the client SHALL set the field *ping_interval*
-to the requested interval in seconds. Otherwise, *ping_interval* MUST
-be set to `0` indicating that no WebSocket *ping* messages SHOULD be
-sent.
+send this message to the server.
+
+* The client MUST set the *your_cookie* field to the cookie the server
+  has used in the nonce of the 'server-hello' message.
+* It SHALL also set the *subprotocols* field to the exact same `Array`
+  of subprotocol strings it has provided to the WebSocket client
+  implementation for subprotocol negotiation.
+* If the user application requests to be pinged (see [RFC 6455 section
+  5.5.2](https://tools.ietf.org/html/rfc6455#section-5.5.2)) in a
+  specific interval, the client SHALL set the field *ping_interval* to
+  the requested interval in seconds. Otherwise, *ping_interval* MUST be
+  set to `0` indicating that no WebSocket *ping* messages SHOULD be
+  sent.
+* If the client has stored the server's public permanent key (32 bytes)
+  and wants to verify it, it MUST include it in the *key* field.
```

Comment:
Mhh. Well, I just wanted to point out that it isn't a key of the client, thus I named it *server_key*. I agree that it is inconsistent. But renaming the other *key* fields would be incompatible to the currently deployed solutions. Accepting both for a short period of time (we haven't even released SaltyRTC yet, so maybe a month) would help but I leave that decision up to you as you have already deployed it.

Regarding the amount of data transmitted... I think a few bytes are negligible (for a signalling protocol).